added more documentation (#43)
JuliaOpt/Dualization.jl: docs/src/index.md

# Dualization.jl Documentation

Dualization.jl is a package written on top of MathOptInterface that allows users to write the dual of a JuMP model automatically.

> This package has two main features, the function dualize that enables users to get a dualized JuMP model and the DualOptimizer that enables users to solve a problem providing the solver it's dual version. 

Reviewer: ```suggestion
This package has two main features: the `dualize` function, which enables users to get a dualized JuMP model, and the `DualOptimizer`, which enables users to solve a problem by providing the solver the dual of the problem. 
```
